2022-03-28 13:05:04,224 INFO    MainThread:39165 [wandb_setup.py:_flush():75] Loading settings from /nethome/mrungta8/.config/wandb/settings
2022-03-28 13:05:04,224 INFO    MainThread:39165 [wandb_setup.py:_flush():75] Loading settings from /nethome/mrungta8/project/ProjectPURE/Mukund/RE_improved_baseline/meta_learning_example_creation/wandb/settings
2022-03-28 13:05:04,224 INFO    MainThread:39165 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-03-28 13:05:04,224 INFO    MainThread:39165 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program_relpath': 'Mukund/RE_improved_baseline/meta_learning_example_creation/train_retacred.py', 'program': 'train_retacred.py'}
2022-03-28 13:05:04,224 INFO    MainThread:39165 [wandb_init.py:_log_setup():386] Logging user logs to /nethome/mrungta8/project/ProjectPURE/Mukund/RE_improved_baseline/meta_learning_example_creation/wandb/run-20220328_130504-2r47q65p/logs/debug.log
2022-03-28 13:05:04,224 INFO    MainThread:39165 [wandb_init.py:_log_setup():387] Logging internal logs to /nethome/mrungta8/project/ProjectPURE/Mukund/RE_improved_baseline/meta_learning_example_creation/wandb/run-20220328_130504-2r47q65p/logs/debug-internal.log
2022-03-28 13:05:04,225 INFO    MainThread:39165 [wandb_init.py:init():420] calling init triggers
2022-03-28 13:05:04,225 INFO    MainThread:39165 [wandb_init.py:init():427] wandb.init called with sweep_config: {}
config: {}
2022-03-28 13:05:04,225 INFO    MainThread:39165 [wandb_init.py:init():471] starting backend
2022-03-28 13:05:04,225 INFO    MainThread:39165 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-03-28 13:05:04,254 INFO    MainThread:39165 [backend.py:ensure_launched():219] starting backend process...
2022-03-28 13:05:04,271 INFO    MainThread:39165 [backend.py:ensure_launched():225] started backend process with pid: 39594
2022-03-28 13:05:04,273 INFO    MainThread:39165 [wandb_init.py:init():480] backend started and connected
2022-03-28 13:05:04,285 INFO    MainThread:39165 [wandb_init.py:init():550] updated telemetry
2022-03-28 13:05:04,334 INFO    MainThread:39165 [wandb_init.py:init():581] communicating current version
2022-03-28 13:05:05,656 INFO    MainThread:39165 [wandb_init.py:init():586] got version response upgrade_message: "wandb version 0.12.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-03-28 13:05:05,656 INFO    MainThread:39165 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-03-28 13:05:05,767 INFO    MainThread:39165 [wandb_init.py:init():624] starting run threads in backend
2022-03-28 13:05:07,878 INFO    MainThread:39165 [wandb_run.py:_console_start():1827] atexit reg
2022-03-28 13:05:07,879 INFO    MainThread:39165 [wandb_run.py:_redirect():1701] redirect: SettingsConsole.REDIRECT
2022-03-28 13:05:07,882 INFO    MainThread:39165 [wandb_run.py:_redirect():1706] Redirecting console.
2022-03-28 13:05:07,885 INFO    MainThread:39165 [wandb_run.py:_redirect():1762] Redirects installed.
2022-03-28 13:05:07,885 INFO    MainThread:39165 [wandb_init.py:init():651] run started, returning control to user process
2022-03-28 13:21:19,483 INFO    MainThread:39165 [wandb_run.py:_atexit_cleanup():1797] got exitcode: 1
2022-03-28 13:21:19,490 INFO    MainThread:39165 [wandb_run.py:_restore():1769] restore
2022-03-28 13:21:22,122 INFO    MainThread:39165 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 991
  total_bytes: 991
}

2022-03-28 13:21:22,431 INFO    MainThread:39165 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 991
  total_bytes: 991
}

2022-03-28 13:21:22,535 INFO    MainThread:39165 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 991
  total_bytes: 991
}

2022-03-28 13:21:22,638 INFO    MainThread:39165 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 991
  total_bytes: 15700
}

2022-03-28 13:21:22,741 INFO    MainThread:39165 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 15700
  total_bytes: 15700
}

2022-03-28 13:21:22,845 INFO    MainThread:39165 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 15700
  total_bytes: 15700
}

2022-03-28 13:21:22,949 INFO    MainThread:39165 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 15700
  total_bytes: 15700
}

2022-03-28 13:21:23,052 INFO    MainThread:39165 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 15700
  total_bytes: 15700
}

2022-03-28 13:21:23,155 INFO    MainThread:39165 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 15700
  total_bytes: 15700
}

2022-03-28 13:21:23,505 INFO    MainThread:39165 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 15700
  total_bytes: 15700
}

2022-03-28 13:21:23,735 INFO    MainThread:39165 [wandb_run.py:_wait_for_finish():1929] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 15700
  total_bytes: 15700
}
local_info {
}

2022-03-28 13:21:25,421 INFO    MainThread:39165 [wandb_run.py:_append_files():2194] logging synced files
